packages/dso-toolkit/src/styles/mixins/_dso-variables.scss

```diff
@@ -55,5 +55,6 @@ $dso-whitebox-small-min-height: 200px;
 $dso-description-bg-color: $dso-info-bg-color;
 $dso-description-border-color: $dso-info-color;
 
-$dso-table-link-color: $grijs-90;
-$dso-table-header-border-color: $grijs-60;
+// verhuisd naar @dso-toolkit/styling/variables/table.scss
```

Comment:
Hier hoeft niet commentaar dat het verhuisd is, mag gewoon verplaatst worden en hier dus ook gewoon weg

Reply:
aangepast